experimental/Schemes/WeilDivisor.jl

```diff
@@ -672,3 +672,63 @@ function order_on_divisor(
   return order_on_divisor(f, I, check=false)
 end
 
+########################################################################
+# Implement the interface for AbsAlgebraicCycle and WeilDivisor for    
+# ToricDivisor
+#
+# Because of the inclusion order of the files, this has to be postponed
+# until here.
+########################################################################
+
+scheme(td::ToricDivisor) = toric_variety(td)
+
+### Construct the Weil divisor associated to `td`.
+@attr WeilDivisor function underlying_cycle(td::ToricDivisor)
+  X = scheme(td)
+  ideal_dict = IdDict{AbsSpec, Ideal}()
+  for U in affine_charts(X)
+    ideal_dict[U] = ideal(OO(U), one(OO(U)))
+  end
+  II = IdealSheaf(X, ideal_dict, check=true) # TODO: Set to false
+  wd = WeilDivisor(X, ZZ, IdDict{IdealSheaf, ZZRingElem}(II => zero(ZZ)))
+  return wd
+end
+
+# Compute the ideal sheaves of the rays of `fan(X)` according to the 
+# "orbit-cone-correspondence", Theorem 3.2.6 in Cox-Little-Schenk.
+#
+# We would like to use Equation (3.2.7) from page 121 for this.
+#@attr Vector{<:IdealSheaf} function _ideal_sheaves_of_rays(X::NormalToricVariety)
+function _ideal_sheaves_of_rays(X::NormalToricVariety; check::Bool=false)
```

Comment:
@lkastner : This is the procedure that caused the trouble. It seems that the code below is producing the correct results and I was aiming at what we talked about. Maybe you see some possibilities for improvements? Or corrections?